install-rhel6-deps-build-openjdk8.sh

```diff
@@ -86,7 +86,7 @@ TARBALL_NAME="\${TARBALL_BASE_NAME}-jdk_\${PLATFORM_VERSION}"
 TARBALL_NAME_JRE="\${TARBALL_BASE_NAME}-jre_\${PLATFORM_VERSION}"
 SOURCE_NAME="\${TARBALL_BASE_NAME}-sources_\${TARBALL_VERSION}"
 
-CLONE_URL=https://hg.openjdk.java.net/jdk8u/jdk8u
+CLONE_URL=https://hg.openjdk.java.net/jdk8u/jdk8u-dev
```

Comment:
We don't want to change the clone URL. We'll only merge this PR once the feature has been merged into jdk8u. So please revert this hunk.